Dashboard/DashboardApp/Application/Components/OneWireIOServer.xml

```diff
@@ -1,5 +1,6 @@
 <Component Model='OneWireIO.OneWireIOServer' Name='OneWireIOServer' src='Components/OneWireIOServer.xml'  Description='Reads OneWire data from the 1-Wire driver in raspbian'>
   <fs>5</fs>
+  <SchedulingOrder>BottomUp</SchedulingOrder>
```

Comment:
Should probably change this to TopDown since that is the default now.

Reply:
Removed it at all as there is no need to configure defaults.